drivers/src/soc_ifc.rs

```diff
@@ -140,6 +141,31 @@ impl SocIfc {
         soc_ifc.cptra_flow_status().read().ready_for_fw()
     }
 
+    /// Set mailbox flow done
+    ///
+    /// # Arguments
+    ///
+    /// * `state` - desired state of `mailbox_flow_done`.
+    ///
+    /// * None
+    pub fn flow_status_set_mailbox_flow_done(&mut self, state: bool) {
+        let soc_ifc = self.soc_ifc.regs_mut();
+        soc_ifc.cptra_flow_status().write(|_| {
+            CptraFlowStatusWriteVal::from(u32::from(soc_ifc.cptra_flow_status().read()))
+                .mailbox_flow_done(state)
+        });
```

Comment:
You can use `.modify` to simplify this:
```suggestion
        soc_ifc.cptra_flow_status().modify(|w| w.mailbox_flow_done(state));
```

Reply:
Nice! Thanks!